.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: "AWS Acceptance Tests"
+
+on:
```

Comment:
Do you plan to add a scheduled cron after running your initial tests?

Reply:
Whatever you think is best. I like nightly for active plugins.

Reply:
I don't mind nightly, but I fear this may become costly to run those every day, while once a week would strike a reasonable balance between costs and safety. We can augment/reduce the frequency if we think it runs too often/not often enough.